config/sync/config_split.config_split.main.yml

```diff
@@ -0,0 +1,19 @@
+uuid: 35befd00-8e63-45fd-9dc8-67908fdc1fcb
+langcode: en
+status: true
+dependencies: {  }
+id: main
+label: Main
+description: 'Main configuration'
+weight: 0
+stackable: false
+no_patching: false
+storage: folder
+folder: ../config/main
+module:
```

Comment:
I think dblog could be enabled in all non-production environments because developers struggle finding logs from Sumologic. But that requires some extra Monolog configuration and I have not yet managed to make it work together with Monolog in Silta. Locally this works:
```
parameters:
  monolog.channel_handlers:
    default:
      handlers:
        - name: 'rotating_file'
          formatter: 'json'
        - name: 'drupal.dblog'
          formatter: 'line'

services:
  monolog.handler.rotating_file:
    class: Monolog\Handler\RotatingFileHandler
    arguments: ['private://logs/debug.log', 10, 'DEBUG']
```
but I don't know why not in Silta.
Just an idea, not blocking acceptance of this PR.

Reply:
True - if we enable `dblog`, we have to disable `updates_log` for now. I agree with that, but let's also await @ragnarkurmwunder 's opinion.

Reply:
> I have not yet managed to make it work together with Monolog in Silta

Note that there are already services defined for Monolog in Silta that will likely override your customizations:

```sh
/var/www/html$ cat web/sites/default/silta.services.yml
parameters:
  monolog.channel_handlers:
     # Log to stderr by default.
     default: ['stream']

services:
  monolog.handler.stream:
    class: Monolog\Handler\StreamHandler
    # The minimum logging level DEBUG = 100
```

Reply:
Not sure if we need to update this part as well as there's currently no `sites/services.yml` in place.

```php
// Load services definition file.
// Note: settings.silta.php defines Monolog services override via silta.services.yml.
// @see: https://github.com/wunderio/charts/blob/master/drupal/files/silta.services.yml
$settings['container_yamls'][] = DRUPAL_ROOT . '/sites/services.yml';
```